ykllvmwrap/build.rs

```diff
@@ -19,11 +19,17 @@ fn main() {
 
     let mut comp = cc::Build::new();
     for cf in cxxflags {
+        if cf.contains("-std=") {
+            // llvm-config contains the -std=c++17 flag by default. We want
```

Comment:
I guess over time the version of C++ targetted by LLVM will change, so maybe "llvm-config contains a -std=c++XY flag, but we want to force a particular version below, so we override it"?

Reply:
Sure. Fixed here: 2e7983b4116ccd2808604acfab4a4eaeb6ed86f5